optimum/exporters/openvino/model_patcher.py

```diff
@@ -6309,6 +6368,60 @@ def __exit__(self, exc_type, exc_value, traceback):
         self._model.img_processor.embeddings.forward = self._model.img_processor.embeddings._orig_forward
 
 
+def sdpa_attention_forward_fixed(
```

Comment:
which part of this method is patched ?

Reply:
patch added for openvino < v2025.3 for which gqa is not supported for sdpa, looks like everything should be working with v2025.3 which soon be out so removed in https://github.com/huggingface/optimum-intel/pull/1406/commits/870b62b8e103798899039a73fb708728f9cd9dd1 and will upgrade the min openvino version instead